app/views/settings/_organisation.html.erb

```diff
@@ -4,28 +4,26 @@
   <caption class="govuk-visually-hidden">Settings</caption>
   <thead class="govuk-table__head govuk-visually-hidden">
     <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header govuk-!-width-one-half">Setting</th>
+      <th scope="col" class="govuk-table__header govuk-!-width-one-third">Setting</th>
       <th scope="col" class="govuk-table__header">Value</th>
       <th scope="col" class="govuk-table__header">Action</th>
     </tr>
   </thead>
   <tbody class="govuk-table__body">
     <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__cell govuk-!-width-one-half govuk-!-font-weight-regular">
+      <th scope="row" class="govuk-table__cell govuk-!-width-one-third">
         Service email
       </th>
       <td class="govuk-table__cell">
         <%= current_organisation.service_email %>
       </td>
       <td class="govuk-table__cell govuk-table__cell--numeric">
-        <%= link_to edit_organisation_path(current_organisation.id), class: "govuk-link govuk-link--no-visited-state" do %>
-          Change <span class='govuk-visually-hidden'>service email</span>
-        <% end %>
+        <%= link_to "Change", edit_organisation_path(current_organisation.id), class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
Similar to my previous comment on visually hidden text, I'd want this to be kept here, rather than removed.

Reply:
Likewise.